nautobot/cloud/filters.py

```diff
@@ -122,8 +122,8 @@ class CloudServiceFilterSet(NautobotFilterSet):
             "cloud_account__description": "icontains",
             "cloud_network__name": "icontains",
             "cloud_network__description": "icontains",
-            "cloud_type__name": "icontains",
-            "cloud_type__description": "icontains",
+            "cloud_resource_type__name": "icontains",
+            "cloud_resource_type__description": "icontains",
```

Comment:
I didn't change it in this PR, but it's inconsistent at best that `CloudServiceFilterSet.q` is *much* more broadly defined than `CloudNetworkFilterSet.q`....

Reply:
It's important that we get the q filter correct since it's used for DynamicModelChoiceFields. I'd say go ahead and add the `cloud_account` and `cloud_resource_type` predicates to the `CloudNetworkFilterSet.q` filter.